deploy/job-manager/deploy.sh

```diff
@@ -20,28 +21,29 @@ function stderr() {
     exit 1
```

Comment:
Would you mind changing the funtion `line()` to 
```bash
printf %"$(tput cols)"s |tr " " "="
```
so that it looks more appealing?